Chorelord/SetupController.swift

```diff
@@ -34,16 +55,70 @@ class SetupViewController: UIViewController, UITextFieldDelegate {
   }
   
   func textFieldDidEndEditing(textField: UITextField) {
-    choreLabel.text = textField.text
+    if let textValue = textField.text {
+      switch (textField.tag) {
+        case TextFieldTags.GroupNameTag.rawValue:
+          tempGroupName = textValue
+          break
+        case TextFieldTags.IntervalTag.rawValue:
+          tempInterval = Int(textValue)
+          break
+        default:
+          break
+      }
+    }
+    textField.resignFirstResponder()
   }
+  
+  func dismissKeyboard() {
+    view.endEditing(true)
+  }
+  
   // MARK: Actions
   @IBAction func addButton(sender: UIButton) {
     let storyboard = UIStoryboard(name: "ChoreList", bundle: nil)
-    let controller = storyboard.instantiateViewControllerWithIdentifier("List") as UIViewController
+    let controller = storyboard.instantiateViewControllerWithIdentifier("List") as! ChoreListViewController
+    controller.updateChoreList(self.tempInterval!, date: self.tempStartDate!)
```

Comment:
`self` here

also if these values are always going to exist, you could add `!` up top instead of `?`